### PR TITLE
[DOCS] Alerts page visualizations, new treemap view

### LINKED ISSUE #2261: [DOCS] Alerts page visualizations, new treemap view
## Description 

PR: https://github.com/elastic/kibana/pull/126896 (lots of details here)

The Alerts page now displays a single visualization pane, with a drop-down to select **Table**, **Trend**, or **Treemap**. And [treemap](https://en.wikipedia.org/wiki/Treemapping) is a new visualization type that displays results as nested, proportionally-sized tiles.

treemap

## Notes

- OK to use simpler language like "nested groups" and "fine tune" instead of "multidimensional," which users might misinterpret as 3D graphs, etc.
- Make sure we carefully explain & distinguish treemap's severity, risk, and color coding, because it appears kind of complicated.
- Caveat: Some treemap tiles can be very tiny, with no visible text.
- Default view is `Trend`.
- Default grouping for Table and Treemap is `Group by: kibana.alert.rule.name` and `Group on top: host.name`. Explain how this works, and suggest other good pairings for meaningful results. PR has some info on "[Recommended groups](https://github.com/elastic/kibana/pull/126896#issuecomment-1180336160). "@paulewing @michaelolo24 might have recommendations too.
- Only appears to be on the Alerts page. Does NOT affect alerts/events tables in other contexts, like Hosts/Network/Users or rule details page | Alerts tab. (Confirm this before release.)

## Docs to update

- [Manage detection alerts](https://www.elastic.co/guide/en/security/master/alerts-ui-manage.html) page
- Alerts page image: [alert-page.png](https://www.elastic.co/guide/en/security/master/detections/images/alert-page.png) (make sure it's single-sourced)
- Consider if a separate sub-topic is needed to explain all the new functionality

## Bonus update!

- The **Manage rules** link on the Alerts page [is now a button](https://github.com/elastic/kibana/pull/138018)! Since I'm updating docs & images for the Alerts page in this issue/PR, we can bundle in this little update, too.
<p align="center">image</p>

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2261.

Previews:
- [Manage detection alerts](https://security-docs_2280.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html) - small revision of "View and filter detection alerts," added link to new subtopic
- [Visualize detection alerts](https://security-docs_2280.docs-preview.app.elstc.co/guide/en/security/master/visualize-alerts.html) - new subtopic

Note: The animation `treemap-click.gif` was captured right before the "Manage rules" link was turned into a button. For now, I'm deprioritizing getting this GIF updated just for that minor detail, since it doesn't affect the functionality being illustrated, and it involves a lot of setup work.

### CODE REVIEW

**Review by andrew-goldstein — APPROVED** [review]
Thanks for writing the new  _Visualize detection alerts_ documentation @joepeeples!

The new content provides an easy-to-digest overview, while also relaying the details when relevant, i.e. _For groupings with a lot of unique values, the top 1,000 results are displayed._

LGTM 🚀

File: docs/detections/alerts-visualizations.asciidoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* *Group by top* (if available): Secondary field for further subdividing grouped alerts.
+
+For example, you can group first by rule name (`Group by: kibana.alert.rule.name`), then by host name (`Group by top: host.name`) to visualize which detection rules generated alerts, and which hosts triggered each of those rules. For groupings with a lot of unique values, the top 1,000 results are displayed.
+
+NOTE: The *Group by top* option isn't available in all view types. You can also leave *Group by top* blank to group by only the primary field in *Group by*.
+
+To reset a view to default settings, click the settings menu (image:images/three-dot-icon.png[Settings menu icon,18,18]) in the upper-right and select *Reset group by fields* (the settings menu appears when hovering over the area).
+
+=== Trend
+The trend view shows the occurrence of alerts over time. This is the default visualization on the Alerts page, and by default it groups alerts by detection rule name (`kibana.alert.rule.name`).
```
Comment (nastasha-solomon) [inline docs/detections/alerts-visualizations.asciidoc:21]:
```suggestion
The trend view is the default visualization on the Alerts page and shows the occurrence of alerts over time. By default, it groups alerts by detection rule name (`kibana.alert.rule.name`).
```

File: docs/detections/alerts-visualizations.asciidoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+NOTE: The *Group by top* menu is unavailable for the trend view.
+
+[role="screenshot"]
+image::images/alerts-viz-trend.png[Trend visualization for alerts]
+
+=== Table
+The table view shows the count of alerts in each group. By default, the table groups alerts first by detection rule name (`kibana.alert.rule.name`), then by host name (`host.name`).
```
Comment (nastasha-solomon) [inline docs/detections/alerts-visualizations.asciidoc:29]:
```suggestion
The table view shows the count of alerts in each group. By default, it groups alerts by detection rule name (`kibana.alert.rule.name`) first, then by host name (`host.name`).
```

File: docs/detections/alerts-visualizations.asciidoc [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+image::images/alerts-viz-treemap.png[Treemap visualization for alerts]
+
+Larger tiles represent more frequent alerts, and each tile's color is based on the alerts' risk score:
+
+* *Green*: Low risk (`0` - `21`)
+* *Yellow*: Medium risk (`22` - `47`)
+* *Orange*: High risk (`48` - `73`)
+* *Red*: Critical risk (`74` - `100`)
+
+By default, the treemap groups alerts first by detection rule name (`kibana.alert.rule.name`), then by host name (`host.name`). This shows which rules generated the most alerts, and which hosts were responsible.
```
Comment (nastasha-solomon) [inline docs/detections/alerts-visualizations.asciidoc:47]:
```suggestion
By default, the treemap groups alerts by detection rule name (`kibana.alert.rule.name`) first, then by host name (`host.name`). This shows which rules generated the most alerts, and which hosts were responsible.
```

**Review by nastasha-solomon — APPROVED** [review]
Just left a few small suggestions. This is a really educational topic - great job on it!

File: docs/detections/alerts-visualizations.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/detections/alerts-visualizations.asciidoc:29]: @nastasha-solomon I'm still leaning toward a more parallel form "first by x, then by y." I also think that "first" kind of gets lost after the long field name in parentheses.
```suggestion
The table view shows the count of alerts in each group. By default, it groups alerts first by detection rule name (`kibana.alert.rule.name`), then by host name (`host.name`).
```

Comment (nastasha-solomon) [reply docs/detections/alerts-visualizations.asciidoc:29]: No strong feelings either way :)

**Review by benironside — APPROVED** [review]
LGTM!

**Comment by samratbhadra-qasource** [comment]
Hi @joepeeples 

We have reviewed the shared preview for **Alerts page visualizations and new treemap view** and all the required change are present and correct. ✔️

**Screenshots:**
alerts-viz-treemap
alert-page

Hence we are adding "QA:Validated" label to it.

Thanks!

**Review by michaelolo24 — APPROVED** [review]
This looks great! Thanks @joepeeples LGTM 🚀

File: docs/detections/alerts-visualizations.asciidoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+[role="screenshot"]
+image::images/alerts-viz-treemap.png[Treemap visualization for alerts]
+
+Larger tiles represent more frequent alerts, and each tile's color is based on the alerts' risk score:
+
+* *Green*: Low risk (`0` - `21`)
+* *Yellow*: Medium risk (`22` - `47`)
+* *Orange*: High risk (`48` - `73`)
+* *Red*: Critical risk (`74` - `100`)
```
Comment (joepeeples) [inline docs/detections/alerts-visualizations.asciidoc:45]:
```suggestion
* *Green*: Low risk (`0` - `46`)
* *Yellow*: Medium risk (`47` - `72`)
* *Orange*: High risk (`73` - `98`)
* *Red*: Critical risk (`99` - `100`)
```

**Comment by joepeeples** [comment]
Updated the risk score ranges for tile colors, per https://github.com/elastic/security-docs/issues/2341 and esp. https://github.com/elastic/kibana/issues/139195#issuecomment-1222918804.

**Comment by samratbhadra-qasource** [comment]
Hi @joepeeples 

We have reviewed the shared preview for Alerts page visualizations and new treemap view and all the required change are present and correct. ✔️

**Screenshots:**

Screenshot (839)

Hence, we are adding "QA:Validated" label to it.

Thanks!

**Pull request merged**